backend/development.ini.sample

```diff
@@ -286,11 +286,15 @@ email.notification.reply_to.email = reply+{content_id}@trac.im
 email.notification.references.email = thread+{content_id}@trac.im
 email.notification.content_update.subject = [{website_title}] [{workspace_label}] {content_label} ({content_status_label})
 email.notification.created_account.subject = [{website_title}] Someone created an account for you
+email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
```

Comment:
That's for an email who inform you about your share with other persons ?

Reply:
yes.

Reply:
ok